File: pbweberogwcl/src/app/disimpegni/commons/dto/esito-salva-revoca-dto.ts
/*
 * Copyright Regione Piemonte - 2024 
 * SPDX-License-Identifier: EUPL-1.2 
*/

/* tslint:disable */
import { MessaggioDTO } from '../dto/messaggio-dto';
export interface EsitoSalvaRevocaDTO {
  esito?: boolean;
  msgs?: Array<MessaggioDTO>;
}

File: pbservizit/pbservizit-jar/src/main/java/it/csi/pbandi/pbservizit/pbandisrv/dto/revoca/EsitoSalvaRevocaDTO.java
/*******************************************************************************
* Copyright Regione Piemonte - 2024
* SPDX-License-Identifier: EUPL-1.2
******************************************************************************/

package it.csi.pbandi.pbservizit.pbandisrv.dto.revoca;

public class EsitoSalvaRevocaDTO implements java.io.Serializable {
	static final long serialVersionUID = 1;
	private java.lang.Boolean esito;
	private MessaggioDTO[] msgs;
	public java.lang.Boolean getEsito() {
		return esito;
	}
	public void setEsito(java.lang.Boolean esito) {
		this.esito = esito;
	}
	public MessaggioDTO[] getMsgs() {
		return msgs;
	}
	public void setMsgs(MessaggioDTO[] msgs) {
		this.msgs = msgs;
	}

	
	
}
